src/components/Kv/KvPillToggle.vue

```diff
@@ -26,45 +26,71 @@ export default {
 	},
 	methods: {
 		pillToggled(key) {
-			this.$emit('pillToggled', key);
+			this.selected = key;
+			this.$emit('pill-toggled', key);
+		}
+	},
+	created() {
+		if (this.options.length > 0) {
+			const option = this.options[0];
+			this.selected = option.key;
 		}
 	}
 };
 
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
 @import "settings";
 
 $form-border-radius: rem-calc(3);
 
-.pill-option {
-	display: inline-block;
-	border: 1px solid $subtle-gray;
-	cursor: pointer;
-}
+.pill-toggle {
+	font-size: rem-calc(14);
+	height: rem-calc(34);
+	width: 100%;
 
-.pill-option:first-child {
-	border-top-left-radius: $form-border-radius;
-	border-bottom-left-radius: $form-border-radius;
-}
+	span {
+		background: $tab-pill-background;
+		border: $tab-pill-border;
+		color: $tab-pill-color;
+		cursor: pointer;
+		display: inline-block;
+		height: 100%;
+		min-width: rem-calc(85);
 
-.pill-option:nth-child(2) {
-	border-top-right-radius: $form-border-radius;
-	border-bottom-right-radius: $form-border-radius;
-}
+		.title {
+			align-items: center;
+			display: flex;
+			height: 100%;
+			justify-content: center;
+			text-align: center;
+			width: 100%;
+		}
 
-.pill-option.active {
-	border: 1px solid $kiva-darkgreen;
-	cursor: default;
-}
+		&:first-child {
+			border-bottom-left-radius: $form-border-radius;
+			border-top-left-radius: $form-border-radius;
+		}
 
-h3 {
-	padding: rem-calc(7) rem-calc(20);
-}
+		&:last-child {
+			border-bottom-right-radius: $form-border-radius;
+			border-top-right-radius: $form-border-radius;
+		}
 
-.pill-option.active h3 {
-	background-color: $kiva-green;
-}
+		&:not(:first-child):not(:last-child) {
```

Comment:
Can we move these lines 82-25 up under line 60 (under selector `.pill-toggle span`)? It'll have the same effect up there.

Reply:
@ksekhri PR updated per your suggestion.